engine/dlib/src/dmsdk/dlib/log.h

```diff
@@ -211,6 +219,46 @@ void dmLogInternal(dmLogSeverity severity, const char* domain, const char* forma
 #define dmLogOnceFatal(format, args... ) dmLogOnceCritical(dmLogFatal, format, ## args )
 #endif
 
+/*# dmLogListener callback typedef
+ *
+ * dmLog listener function type. Provides all logs from dmLog* functions and print/pprint Lua functions.
+ * Used with dmLogRegisterListener() and dmLogUnregisterListener()
+ *
+ * @typedef
+ * @name dmLogListener
+ * @param severity [type:dmLogSeverity]
+ * @param domain [type:const char*]
+ * @param formatted_string [type:char*]
+ */
+typedef void (*dmLogListener)(dmLogSeverity severity, const char* domain, char* formatted_string);
```

Comment:
I think "const char* formatted_string" is better, to communicate that the user shouldn't modify the buffer.

Reply:
thx!